examples/refrigerator-app/silabs/include/RefrigeratorManager.h

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ *    Copyright (c) 2019 Google LLC.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+#include "AppEvent.h"
+#include <app/clusters/refrigerator-alarm-server/refrigerator-alarm-server.h>
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/ConcreteAttributePath.h>
+#include <app/util/attribute-storage.h>
+#include <lib/support/logging/CHIPLogging.h>
+#include <examples/refrigerator-app/refrigerator-common/include/static-supported-temperature-levels.h>
```

Comment:
Maybe you should replace this by:
```#include <../../refrigerator-common/include/static-supported-temperature-levels.h>```

Otherwise build fails with gn_silabs_example.sh script:
```
./scripts/examples/gn_silabs_example.sh ./silabs_examples/refrigerator-app/silabs/ ./out/refrigerator-app BRD2703A

../../../examples/refrigerator-app/silabs/include/RefrigeratorManager.h:27:10: fatal error: examples/refrigerator-app/refrigerator-common/include/static-supported-temperature-levels.h: No such file or directory
   27 | #include <examples/refrigerator-app/refrigerator-common/include/static-supported-temperature-levels.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Added the change